wiki/helpers.cjs

```diff
@@ -17,6 +17,7 @@ exports.getFragment = function (func) {
 }
 
 exports.cleanNames = function (names) {
+    if (!names) return []
```

Comment:
@olegtaranenko can you try again after this commit? adding this line seems to have fixed it